pages/common/git-checkout.md

```diff
@@ -4,12 +4,16 @@
 
 - Switch to another branch:
 
-`git checkout {{BRANCH-NAME}}`
+`git checkout {{branch_name}}`
 
 - Create and switch to a new branch:
 
-`git checkout -b {{BRANCH-NAME}}`
+`git checkout -b {{branch_name}}`
 
 - Undo unstaged local modification:
 
 `git checkout .`
+
+- Checkout a file from another branch:
```

Comment:
I would refrain from using the word "checkout" as we are trying to explain what "checkout" means in the first place. How about - "Replace the contents of {{file_name}} in the current branch with {{branch_name}}" ? Or something on those lines ..